src/edtr-io/plugins/highlight/editor.tsx

```diff
@@ -90,4 +45,42 @@ export function HighlightEditor(props: HighlightProps) {
       code={state.code.value}
     />
   )
+
+  function renderSettings() {
+    return (
+      <div className="mt-2 flex justify-between">
+        <label>
+          Language:{' '}
+          <select
+            onChange={(e) => state.language.set(e.target.value)}
+            className="cursor-pointer"
+          >
+            {languages.map((language) => {
+              return (
+                <option
+                  key={language}
+                  value={language}
+                  selected={
```

Comment:
info, this is the warning I get about selected:

![grafik](https://user-images.githubusercontent.com/13507950/223726292-9a237d9f-aee6-4ab7-9789-16e0f50efe03.png)

I don't have a strong opinion here, but with a slight preference to remove the react warning

Reply:
will change. Where did you catch that?

Reply:
console of localhost, the first time I visit the page